configure.ac

```diff
@@ -343,6 +343,7 @@ SR_DRIVER([Norma DMM], [norma-dmm], [serial_comm])
 SR_DRIVER([OpenBench Logic Sniffer], [openbench-logic-sniffer], [serial_comm])
 SR_DRIVER([PCE PCE-322A], [pce-322a], [serial_comm])
 SR_DRIVER([Pipistrello-OLS], [pipistrello-ols], [libftdi])
+SR_DRIVER([RaspberryPI PICO], [raspberrypi-pico])
```

Comment:
@pico-coder Based on comment on dependency on serial port in "Conversation" here should be:
`SR_DRIVER([RaspberryPI PICO], [raspberrypi-pico], [serial_comm])`

I have compiled libsigrok with your driver about month ago and I needed to add that dependency.

Reply:
I was able to compile it without this change just fine